src/html/shared/swiper.html

```diff
@@ -0,0 +1,58 @@
+<div class="c-swiper">
+    <div class="o-wrapper">
+        <div class="c-swiper__header">
+            <h1>Why choose Shopiflight?</h1>
+        </div>
+
+        <div class="swiper-container u-hide@tablet">
+            <div class="swiper-wrapper">
+                <div class="swiper-slide">
+                    <div class="c-swiper__content">
+                        <button class="c-icon  c-icon--huge  c-icon--static">{{sprite("feature")}}</button>
+                        <h3>Feature 1</h3>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    </div>
+                </div>
+                <div class="swiper-slide">
+                    <div class="c-swiper__content">
+                        <button class="c-icon  c-icon--huge  c-icon--static">{{sprite("feature")}}</button>
```

Comment:
@xavierang these are likely not going to allow user interaction so could be `<span>`'s instead of `<button>`'s